### PR TITLE
build: update renamed codegen types to be backwards-compatible

### PR DESCRIPTION
**Pull request by bertrmz**
### Overview of Changes in this PR

-   update renamed codegen types to be backwards-compatible

This unblocks the workflow generating Go types from the Passage SDK spec.

### CODE REVIEW

File: generate.sh [hunk 1]
```diff
@@ -28,7 +28,8 @@ transforms='{
     "CreateMagicLinkRequest": "CreateMagicLinkBody",
     "CreateUserRequest": "CreateUserBody",
     "Inactive": "StatusInactive",
-    "Login": "LoginType",
+    "MagicLinkTypeLogin": "LoginType",
+    "MagicLinkTypeVerifyIdentifier": "VerifyIdentifierType",
```
Comment (ctran88) [inline generate.sh:32]: i got an error that this type didn't exist:
```
curl -o openapi.json https://api.swaggerhub.com/apis/passage/passage-mgmt-sdk-api/1

./generate.sh openapi.json

Renamed 2 occurrences in 2 files in 2 packages.
Renamed 4 occurrences in 3 files in 2 packages.
Renamed 8 occurrences in 3 files in 2 packages.
Renamed 2 occurrences in 2 files in 2 packages.
Renamed 2 occurrences in 2 files in 2 packages.
Renamed 1 occurrence in 1 file in 1 package.
Renamed 4 occurrences in 1 file in 1 package.
Renamed 2 occurrences in 2 files in 2 packages.
Renamed 1 occurrence in 1 file in 1 package.
Renamed 1 occurrence in 1 file in 1 package.
Renamed 5 occurrences in 3 files in 2 packages.
Renamed 9 occurrences in 3 files in 2 packages.
Renamed 1 occurrence in 1 file in 1 package.
Renamed 1 occurrence in 1 file in 1 package.
Renamed 1 occurrence in 1 file in 1 package.
Renamed 1 occurrence in 1 file in 1 package.
Renamed 1 occurrence in 1 file in 1 package.
Renamed 1 occurrence in 1 file in 1 package.
gorename: package "github.com/passageidentity/passage-go" has no member "VerifyIdentifierType"
```

Comment (bertrmz) [reply generate.sh:32]: It's in the codegen file here: https://github.com/passageidentity/passage-go/blob/e1b7a15873987993c91d258c6bdee5fc67d35bf7/passage.gen.go#L112

Comment (ctran88) [reply generate.sh:32]: Yeah I do see it in my unmodified codegen file. Is it perhaps erroring on https://github.com/passageidentity/passage-go/blob/c80a7a4a8affcc52620ad4a436d6fc1a33cefd18/generate.sh#L45

Comment (bertrmz) [reply generate.sh:32]: Ah yeah, this is updated in https://github.com/passageidentity/passage-go/pull/91/commits/4485ee26749780493edc48bbd3528ae1705fb607.

**Review by ctran88 — APPROVED** [review]

**Pull request merged**